### DISCUSSION TITLE
Unable to parse latex using rehypeKatex plugin

### DISCUSSION THREAD

**Discussion by agrajgarg** [remarkjs/remark#1353] [Q&A]
`{
                            text.map((message, index) => (
                                <div key={index} className="markdown" id="bot-msg">
                                    <Markdown
                                        remarkPlugins={[remarkMath]} 
                                        rehypePlugins={[rehypeRaw, rehypeKatex]}
                                        components={{
                                            img: ({node, ...props}) => <img style={{maxWidth: '100%'}} {...props} />
                                        }}
                                    >
                                        {preprocessLaTeX(message)}
                                    </Markdown>
                                </div>
                            ))
                        }`
                        
  Here i am using rehypeKatex to render latex text returned by gpt. I know that gpt returns latex equations in \\[] but katex supports latex equations starting with $$. I used the following method to convert the text in the correct format 
  
  `const preprocessLaTeX = (content) => {
        debugger;
        // Replace block-level LaTeX delimiters \[ \] with $$ $$
        const blockProcessedContent = content.replace(
          /\\\[(.*?)\\\]/gs,
          (_, equation) => `$$${equation}$$`,
        );
        // Replace inline LaTeX delimiters \( \) with $ $
        const inlineProcessedContent = blockProcessedContent.replace(
          /\\\((.*?)\\\)/gs,
          (_, equation) => `$${equation}$`,
        );
        return inlineProcessedContent;
    };`
    
    But I am getting this error constantly
    
    **_propertyInformation.hastToReact is undefined** 
    
    Can anyone help?
    Tried all methods.
    
    I am using parcel bundler to build my react app

**Comment by wooorm** [comment]
Seems like a duplicate of https://github.com/remarkjs/react-markdown/issues/851
